src/js/utils/browser.ts

```diff
@@ -1,16 +1,16 @@
+declare const __FLASH_VERSION__: number;
+
 const userAgent = navigator.userAgent;
 
-function userAgentMatch(regex) {
+function userAgentMatch(regex: RegExp): boolean {
     return userAgent.match(regex) !== null;
 }
 
-function lazyUserAgentMatch(regex) {
-    return function () {
-        return userAgentMatch(regex);
-    };
+function lazyUserAgentMatch(regex: RegExp): () => boolean {
+    return () => userAgentMatch(regex);
 }
```

Comment:
It looks like for 'Function' return types, they can be more explicitly typed like above using arrow functions or more simply typed like below:

> `function lazyUserAgentMatch(regex: RegExp): Function`

I want to note that I am not 100% as to the larger difference between the two aside from being more or less explicit with types.